.circleci/config.yml

```diff
@@ -1448,6 +1448,30 @@ workflows:
             - attach_workspace:
                 at: /
 
+  GCP-Weekly-Services-HTS-Restart-Performance-15N-15C:
+    triggers:
+      - schedule:
+          cron: "5 15 * * 6,0"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - build-platform-and-services
+      - jrs-regression:
+          name: "GCP-Weekly-Services-HTS-Restart-Performance-15N-15C"
+          context: Slack
+          regression_path: /swirlds-platform/regression
+          result_path: results/15N_15C/
+          config_file: "configs/services/suites/weekly/GCP-Weekly-Services-HTS-Restart-Performance-15N-15C.json"
+          workflow-name: "GCP-Weekly-Services-HTS-Restart-Performance-15N-15C"
```

Comment:
I thought we already simplified the name, didn't we? If not, we should use only `name: "GCP-Weekly-Services-HTS-Restart-Performance-15N-15C"` and maybe `config_type: "weekly"`.

Reply:
@qnswirlds We have simplified it before moving to the summary history changes. When we used `jrs_regression` job , we did not simplify it. 
We thought it might be helpful to leave it this way, as now all developers can run tests on their branches with simple changes to those parameters apart from automated nightly runs.

Reply:
@Neeharika-Sompalli, I would disagree. Simplify the config would also make devs' life easier. For example, I could use `name: "my_personal_json"` with `config_type: "personal"` without copying around the `my_personal_json` part. The `workflow-name` will be the same (automatically) as `name` and `config_file` will be automatically `configs/services/suites/personal/my_personal_json`. That will save time and avoid unnecessary copy-paste mistakes.

Reply:
@qnswirlds are you suggesting to remove the `config_file` parameter here?

Reply:
It should be simplified into `config_type` as mentioned.

Reply:
So we need to have the workflow name to be exactly the same as json file name, I am ok with this.

Reply:
Fixed. Thanks @qnswirlds @ljianghedera . Passed test after the changes : https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1612382576152000